### PR TITLE
Made a function that asks how many times the lights flashed

### PR DESCRIPTION
**Pull request by e093**
After flashing, the computer will prompt the user to input the amount of times the red light and the green light flashed, and replies whether the answers are correct or not.

### CODE REVIEW

File: Communication_test.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            time.sleep(1)
+            redcount += 1
+        else:
+            red.toggle()
+            time.sleep(0.5)
+            red.toggle()
+            time.sleep(1)
+            greencount += 1
+    redcountans = int(input"How many times did the red light flash?")
+    if redcountans == redcount:
```
Comment (DrOrnage) [inline Communication_test.py:25]: There is no need for the program to do anything if the user input is correct.

File: Communication_test.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        print("You're correct!")
+    else:
+        print("You're wrong!")
```
Comment (DrOrnage) [inline Communication_test.py:29]: the result of a wrong input should be that this character is shown in light signals again, not an output saying you are wrong.
```suggestion
        pass
```

**Review by DrOrnage — CHANGES_REQUESTED** [review]
Please make changes according to my suggestions.

File: Communication_test.py (hunk 2 above)
Comment (e093) [reply Communication_test.py:29]: Didn't thought of that — thanks for the suggestion! I'll work on it.

File: Communication_test.py (hunk 1 above)
Comment (e093) [reply Communication_test.py:25]: No problem, I will have this feature removed :)

**Comment by DrOrnage** [comment]
closed

**Pull request closed without merging**